cli/lsp/logging.rs

```diff
@@ -0,0 +1,54 @@
+// Copyright 2018-2021 the Deno authors. All rights reserved. MIT license.
+
+use std::sync::atomic::AtomicBool;
+use std::sync::atomic::AtomicUsize;
+use std::sync::atomic::Ordering;
+
+static LSP_DEBUG_FLAG: AtomicBool = AtomicBool::new(false);
+static LSP_LOG_LEVEL: AtomicUsize = AtomicUsize::new(log::Level::Info as usize);
+
+pub fn set_lsp_debug_flag(value: bool) {
+  LSP_DEBUG_FLAG.store(value, Ordering::SeqCst)
+}
+
+pub fn lsp_debug_enabled() -> bool {
+  LSP_DEBUG_FLAG.load(std::sync::atomic::Ordering::SeqCst)
+}
+
+pub fn set_lsp_log_level(level: log::Level) {
+  LSP_LOG_LEVEL.store(level as usize, Ordering::SeqCst)
+}
+
+pub fn lsp_log_level() -> log::Level {
+  let level = LSP_LOG_LEVEL.load(std::sync::atomic::Ordering::SeqCst);
```

Comment:
```suggestion
  let level = LSP_LOG_LEVEL.load(Ordering::SeqCst);
```

Reply:
Thanks! I had these in the macro previously while it was a WIP and forgot to unqualify them when I moved it to a function.